provider/search-provider/src/main/java/dev/ikm/tinkar/provider/search/Searcher.java

```diff
@@ -344,4 +346,25 @@ public static List<PublicId> getAllowedResultsFromResultConformance(NavigationCa
         return allowedResultsList;
     }
 
+    /**
+     * Returns List of PublicIds for the Concepts tagged with the Membership Pattern
+     *
+     * @param   memberPatternId PublicId of the Membership Pattern
+     * @return  List of PublicIds for the Concepts tagged with the Membership Pattern
+     */
+    public static List<PublicId> membersOf(PublicId memberPatternId) {
+        List<PublicId> conceptIds = new ArrayList<>();
+        try {
+            EntityService.get().getEntity(EntityService.get().nidForPublicId(memberPatternId)).ifPresent((e) -> {
```

Comment:
@dettmoney suggest using one of the other overloaded methods for getEntity to reduce the number of calls to EntityService.